test/spec/modules/imuIdSystem_spec.js

```diff
@@ -122,6 +137,7 @@ describe('imuId module', function () {
     it('should return the undefined when success response', function () {
       const res = apiSuccessProcess({
         uid: 'test',
+        ppid: 'test',
```

Comment:
you might want to add a test that it can successfully get set as the gam ppid

Reply:
maybe next to this one https://github.com/prebid/Prebid.js/blob/d46661a4aa8994d57f3bea3055bd4e4c8aa50e10/test/spec/modules/userId_spec.js#L402

Reply:
Thanks for the review.
Added "imppid" spec.